awx/main/wsrelay.py

```diff
@@ -191,7 +191,7 @@ async def run_producer(self, name, websocket, group):
                         return
 
                     continue
-                except aioredis.errors.ConnectionClosedError:
+                except redis.exceptions.ConnectionError:
```

Comment:
this might not be the right 1 to 1 replacement of the exception

Reply:
This was the exception I saw used in the aioredis code base where the ConnectionClosedError was before but I'm open to other options

Reply:
this should be ok